Xamarin.Forms.Platform.Android/Renderers/CheckBoxRenderer.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.ComponentModel;
+using Android.Content;
+using Android.Widget;
+using ACheckBox = Android.Widget.CheckBox;
+
+namespace Xamarin.Forms.Platform.Android
+{
+	public class CheckBoxRenderer : ViewRenderer<CheckBox, ACheckBox>, CompoundButton.IOnCheckedChangeListener
```

Comment:
Probably won't include this one. We're not really adding new controls behavior to non appcompat. It's mainly just in maintenance mode

Reply:
Now addressed!